documentation/modules/ROOT/pages/configuration/content-based-routing.adoc

```diff
@@ -33,27 +33,31 @@ The {prodname} content-based routing SMT is a Technology Preview feature. Techno
+For example:
 
 * It is necessary to compile the transformation up front and deploy it to Kafka Connect.
 * Every change needs code recompilation and redeployment, leading to inflexible operations.
 
 The content-based routing SMT supports scripting languages that integrate with https://jcp.org/en/jsr/detail?id=223[JSR 223] (Scripting for the Java(TM) Platform).
 
+{prodname} does not come with any implementations of the JSR 223 API.
+To use an expression language with {prodname}, you must download the JSR 223 script engine implementation for the language, and add to your {prodname} connector plug-in directories, along any other JAR files used by the language implementation.
+For example, for Groovy 3, you can download its JSR 223 implementation from https://groovy-lang.org/. 
+The JSR 223 implementation for GraalVM JavaScript is available at https://github.com/graalvm/graaljs.
+
+// Type: procedure
+// Title: Setting up the {prodname] content-based-routing SMT
+// ModuleID: setting-up-the-debezium-content-based-routing-smt
 [[set-up-content-based-routing]]
 == Set up
 
-For security reasons, the content-based routing SMT is not part of the {prodname} connector archives,
-but is provided in a separate artifact, debezium-scripting-{debezium-version}.tar.gz, which must be explicitly added to your Kafka Connect environment.
-In order to use this SMT for a given connector plug-in, add the contents of debezium-scripting-{debezium-version}.tar.gz to the connector's plug-in directory.
+For security reasons, the content-based routing SMT is not included with the {prodname} connector archives.
+Instead, it is provided in a separate artifact, `debezium-scripting-{debezium-version}.tar.gz`.
+To use the content-based routing SMT with a {prodname} connector plug-in, you must explicitly add the SMT artifact to your Kafka Connect environment.
 
-Note that this allows anyone who can set up connectors in your Kafka Connect instance to execute scripting expressions.
-You should therefore only add this SMT after securing your Kafka Connect instance and its configuration interface appropriately.
-
-{prodname} itself doesn't come with any implementations of the JSR 223 API.
-To use an expression language, you must download the JSR 223 script engine implementation for the language, and add to your {prodname} connector plug-in directories, along any other JAR files used by the language implementation.
-Eg. for Groovy 3, you can download its JSR 223 implementation from https://groovy-lang.org/. For GraalVM JavaScript, this is is available at https://github.com/graalvm/graaljs.
+IMPORTANT: After the filter SMT is present in a Kafka Connect instance, any user who is allowed to add a connector to the instance can run scripting expressions.
+To ensure that scripting expressions can be run only by authorized users, be sure to secure the Kafka Connect instance and its configuration interface before you add the filter SMT.
```

Comment:
```suggestion
To ensure that scripting expressions can be run only by authorized users, be sure to secure the Kafka Connect instance and its configuration interface before you add the routing SMT.
```

Reply:
Oh! Bad copy/paste. Sorry and thanks for catching it!